csharp/Ice/Callback/Client/Program.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) ZeroC, Inc.
+
+// Slice module EarlyRiser in AlarmClock.ice maps to C# namespace EarlyRiser.
+using EarlyRiser;
+
+// Create an Ice communicator to initialize the Ice runtime.
+using Ice.Communicator communicator = Ice.Util.initialize(ref args);
+
+// Create an object adapter that listens for incoming requests and dispatches them to servants.
+Ice.ObjectAdapter adapter = communicator.createObjectAdapterWithEndpoints("AlarmClockAdapter", "tcp -p 10000");
+
+// Register the MockAlarmClock servant with the adapter, and get an alarm clock proxy.
+// The identity of this Ice object is a UUID, which is suitable for transient objects.
```

Comment:
On second thought, it would better to use a fixed identity like alarmClock. We no longer advocate for globally unique identities.